src/lib/Polykey.ts

```diff
@@ -17,21 +18,22 @@ class Polykey {
     polykeyPath: string = `${os.homedir()}/.polykey`,
     fileSystem: typeof fs,
     keyManager?: KeyManager,
-    vaultManager?: VaultManager,
     peerManager?: PeerManager,
+    vaultManager?: VaultManager,
   ) {
     this.polykeyPath = polykeyPath;
 
     // Set key manager
     this.keyManager = keyManager ?? new KeyManager(this.polykeyPath, fileSystem);
 
-    // Set or Initialize vaultManager
-    this.vaultManager = vaultManager ?? new VaultManager(this.polykeyPath, fileSystem, this.keyManager);
-
     // Initialize peer store and peer discovery classes
-    this.peerManager = peerManager ?? new PeerManager(this.polykeyPath, fileSystem, this.keyManager, this.vaultManager);
+    this.peerManager = peerManager ?? new PeerManager(this.polykeyPath, fileSystem, this.keyManager);
+
+    // Set or Initialize vaultManager
+    this.vaultManager =
+      vaultManager ?? new VaultManager(this.polykeyPath, fileSystem, this.keyManager, this.peerManager);
```

Comment:
If we call in the bin commands `--store`, then we should standardize the name in the variables across the codebase. So `--path` is fine too. But then the idea is `polykeyPath` should be reused everywhere. I think `--store` could be used, it doesn't seem to conflict with anything.